<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QWidget" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelUser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SELAMAT DATANG USER !</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUser_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Halo User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelUser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>471</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboUrutkan">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nama (A-Z)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tanggal (Terbaru)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jumlah (Terbanyak)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Urutkan Berdasarkan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
